services/frontend-v3/src/components/layouts/aboutLayout/AboutLayout.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import PropTypes from "prop-types";
+import AboutLayoutView from "./AboutLayoutView";
+
+/**
+ *  LandingLayout(props)
+ *
+ *  this is the controller for LandingLayoutView
+ */
+const AboutLayout = (type) => {
```

Comment:
Should be deconstructing the prop
```suggestion
const AboutLayout = ({ type }) => {
```